contracts/collections/src/test/tests.rs

```diff
@@ -523,3 +526,89 @@ fn uri_should_fail_when_none_set() {
 
     assert_eq!(client.try_uri(&1), Err(Ok(ContractError::NoUriSet)))
 }
+
+#[test]
+fn should_transfer_when_sender_is_operator() {
+    let env = Env::default();
+    env.mock_all_auths();
+
+    let admin = Address::generate(&env);
+    let operator = Address::generate(&env);
+    let user_a = Address::generate(&env);
+    let user_b = Address::generate(&env);
+
+    let client = initialize_collection_contract(&env, Some(&admin), None, None);
+
+    client.mint(&admin, &user_a, &1, &1);
```

Comment:
nit: `user_a` should be the admin
```suggestion
    let operator = Address::generate(&env);
    let user_a = Address::generate(&env);
    let user_b = Address::generate(&env);

    let client = initialize_collection_contract(&env, Some(&user_a), None, None);

    client.mint(&user_a, &user_a, &1, &1);
```